src/App.tsx

```diff
@@ -1,20 +1,16 @@
-import reactLogo from '@/assets/react.svg';
-
-import { HeaderComponent } from './components/header/Header.component.tsx';
+import AboutComponent from '@/components/about/About.component.tsx';
+import FooterComponent from '@/components/footer/Footer.component.tsx';
+import HeaderComponent from '@/components/header/Header.component.tsx';
 
 import './App.css';
 
 function App() {
     return (
-        <>
+        <div className="about">
```

Comment:
it's not `about` component. Why we have class for main component `about`?

Reply:
fixed class name